addons/functions/functions/fnc_vehicleAction.sqf

```diff
@@ -0,0 +1,123 @@
+#include "script_component.hpp"
+
+params ["_vehicle"];
+
+if (!(isClass (configFile >> "CfgDigVehicles" >> typeOf _vehicle))) exitWith {
+    diag_log "Vehicle Digging not supported";
+};
+
+if (_vehicle getVariable ["GRAD_trenches_vehicleDiggingActionAdded", false]) exitWith {};
```

Comment:
as this is only it should never happen that this function is called twice on the same vehicle, right? 

so if this function *is* being called more than once on the vehicle, something is off.

```suggestion
if (_vehicle getVariable ["GRAD_trenches_vehicleDiggingActionAdded", false]) exitWith {
    WARNING_1("attempt to add vehicle action more than once on %1", _vehicle);
};
```

Reply:
it did, iirc i fixed the cause (classeventhandler was added for all classes or sth)